blog/2022-07-20-centralized-kubectl-access/index.md

```diff
@@ -0,0 +1,80 @@
+
+We installed Paralus on AKS, so we created a new cluster and installed paralus using the following commands
+
+`helm repo add paralus https://paralus.github.io/helm-charts`
+
+```bash
+helm install myrelease paralus/ztka \
+ -f https://raw.githubusercontent.com/paralus/helm-charts/main/examples/values.dev-generic.yaml \
+ --set fqdn.domain="chartexample.com" \
+ -n paralus \
+ --create-namespace
+```
+
+Make sure to update the `fqdn.domain` value with the actual domain that you’d be using. Once the installation is complete, you can configure the DNS records and access the dashboard.
+
+### Importing A Cluster
+
+Once you are within the dashboard, you can start by creating a new project. Within the project, you can import a new cluster. You can follow the below steps to import an existing on prem cluster or any other cloud cluster:
+
+- Click on Import Cluster
+- In the new dialog,
+  - choose `Data Center/Edge` as the environment and `Other` as Kubernetes distribution. Use this for importing a cluster from your onPrem or local host
+
+    <img src="/img/docs/kind-import.gif" alt="Importing a local/onprem Cluster" height="70%" width="70%"/>
+
+  - choose `Public Cloud` as the environment and `GKE` as Kubernetes distribution
+
+    <img src="/img/docs/gke-import.gif" alt="Importing a GKE Cluster" height="70%" width="70%"/>
+
+- Provide a name and description and continue
+- Leave the general and advanced settings as is and move to the final stage
+- Download the generated `bootstrap.yaml` file
+- Apply this file in the cluster that you want to import to Paralus `kubectl apply -f bootstrap.yaml`
+- After the relay agent is up and running, your cluster will be successfully imported to Paralus.
+
+Once imported, you can navigate to the project and view the clusters. You can test if the clusters are configured correctly or not, you can execute kubectl commands from the prompt.
+
+### Configuring Groups & Users
+
+The next step is to create a new Group and assign this project to the group. It’s easier to manage permissions and roles via groups rather than doing it for every individual user.  
+
+So create a group, add the project and choose the permissions. We can choose `project_readonly` to provide only read only permissions or anything else based on your use case.
+
+<img src="/img/docs/groups-users-add.png" alt="Adding user to the group" height="70%" width="70%"/>
+
+Once configured, you can ask the users to re login to verify the access. The users assigned to the project will be able to access the clusters in read only mode and will not be able to create, edit or delete any resource in the cluster.
+
+<img src="/img/docs/roles-new-user-project-final.png" alt="Adding user to the group" height="70%" width="70%"/>
```

Comment:
in this blog post, we choose 'project_readonly' as an example however this image shows project_admin chosen. I understand this is fine since we have mentioned "or anything else based on your use case" but just thought of highlighting this as it may be confusing for few.

Reply:
Done, updated the image.